src/MvvmBlazor/ViewModel/ViewModelBase.cs

```diff
@@ -28,6 +41,24 @@ public virtual void OnPropertyChanged(string propertyName)
             PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
         }
 
+        protected void Subscribe<T>(Expression<Func<T>>? expression, Action<T> action)
+        {
+            var member = expression?.Body as MemberExpression;
+            var propInfo = member?.Member as PropertyInfo;
+            if (propInfo == null)
+            {
```

Comment:
I'm not sure whether this should just return in this case. Throwing a proper error should be better, especially if the property does not exist.

Reply:
That makes sense. Which exception would you propose I throw?

Reply:
There already exists a `BindingException` that could be used here.

Reply:
Updated